Wrappers/Python/cil/optimisation/operators/ProjectionMap.py

```diff
@@ -63,17 +67,50 @@ def __init__(self, domain_geometry, index, range_geometry=None):
                                             range_geometry=range_geometry)
 
     def direct(self,x,out=None):
-
+        r"""
+        Returns the ith (`param_index`) element of the Block data container, :math:`x`
+        
+        Parameters
+        ----------
+        x: `BlockDataContainer`
+        
+        out: `DataContainer`, default `None`
+            If `out` is not `None` the output of the `ProjectionMap` will be filled in `out`, otherwise a new object is instantiated and returned.
+        
+        Returns
+        --------
+        `DataContainer`
+        
+        """
+        
+    
+    
         if out is None:
             return x[self.index].copy()
         else:
             out.fill(x[self.index])
 
     def adjoint(self,x, out=None):
-
+        r"""
+        Returns a `BlockDataContainer` of zeros with the ith (`param_index`) filled with the `DataContainer`, :math:`x`
```

Comment:
Returns a `BlockDataContainer` of zeros with the ith (`index`) filled with the `DataContainer`, :math:`x` ?

Reply:
Thanks for the spot!